f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_funding_request_fs_form.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?><database>
     <element label="Funding Request FS Form" max_length="40" name="x_g_dis_atat_funding_request_fs_form" type="collection">
         <element label="FS Form 7600A Attachment" max_length="40" name="fs_form_7600a_attachment" type="file_attachment"/>
+        <element label="FS Form 7600A Filename" max_length="256" name="fs_form_7600a_filename" type="string"/>
         <element label="FS Form 7600B Attachment" max_length="40" name="fs_form_7600b_attachment" type="file_attachment"/>
+        <element label="FS Form 7600B Filename" max_length="256" name="fs_form_7600b_filename" type="string"/>
```

Comment:
I know we previously had the `filename` column, but just wondering is this so we can provide a standard file name for the attachment despite what the user uploads as the filename?

Reply:
@robertmccardell-ccpo Thanks for confirming!